crates/biome_js_formatter/src/js/lists/parameter_list.rs

```diff
@@ -103,3 +96,37 @@ impl Format<JsFormatContext> for FormatJsAnyParameterList<'_> {
         }
     }
 }
+
+fn join_parameter_list<S>(
+    joiner: &mut JoinNodesBuilder<'_, '_, S, JsFormatContext>,
+    list: &AnyJsParameterList,
+    trailing_separator: TrailingSeparator,
+) -> FormatResult<()>
+where
+    S: Format<JsFormatContext>,
```

Comment:
```suggestion
fn join_parameter_list<S, C>(
    joiner: &mut JoinNodesBuilder<'_, '_, S, C>,
    list: &AnyJsParameterList,
    trailing_separator: TrailingSeparator,
) -> FormatResult<()>
where
    C: JsFormatContext,
    S: Format<C>,
```

Reply:
hmm, it doesn't seem to like this because JsFormatContext is a `struct` and not a `trait`, and trying to use just `FormatContext` or `CstFormatContext` causes a type error in the body of the function, so i'm not sure how to handle that appropriately. since this is just a private function and used inside the js formatter, i think it's probably okay to have it just be hardcoded to `JsFormatContext` rather than being a generic?

Not sure if there's maybe another way to do that i'm unaware of.

Reply:
Ooooh you're right! Then we can discard the suggestion :)